src/language/en-US.json

```diff
@@ -560,9 +561,9 @@
     "show_expired_namespaces": "Show expired namespaces",
     "show_wizard": "Show wizard",
     "sidebar_item_accounts": "Accounts",
+    "sidebar_item_harvesting": "Harvest",
```

Comment:
- Language key-value missing in Jap and CHN
- Language JSON should be sorted by key

Reply:
Fixed